hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/BackendEntityView.avdl

```diff
@@ -24,6 +24,9 @@ protocol BackendEntityViewProtocol {
 
     long duration_millis = 0;
 
+    // duration micro
+    double duration_micros = 0;
```

Comment:
This can be long?

Reply:
As per our discussion, we went with double to store all the significant digits(might need in future). Making it LONG will defeat this purpose.

Reply:
Long's upper limit is 9223372036854775807, which is equal to 292471.20867753599305 years. Do you think a span's duration will exceed this value?

Reply:
we are talking about significant digits, not the range

Reply:
let's say we have .123 ms that would be 0ms, 123 micro seconds.
Now if we have .1234ms that would be 123 micro seconds(if long), 123.4 micro seconds(if double). Isnt it?

Reply:
Aren't we solving for microsecond granularity? It means the smallest duration we can report is 1µ. So 123.4 µs should be rounded off to 123 µs. This will keep things on the query layer simples as you wouldn't have to deal with converting data types.

Reply:
As per discussion with team and tech specs I have added datatype as DOUBLE